using-mobile-extensions/adobe-media-analytics/media-api-reference.md

```diff
@@ -834,7 +834,7 @@ Method to track media events.
 | Variable Name | Description |
 | :--- | :--- |
 | `event` | [Media event](media-api-reference.md#media-events) |
-| `info` | For an `AdBreakStart` event, the `adBreak` information is created by using a [Create AdBreak Object](media-api-reference.md#create-adbreak-object).  For an `AdStart` event, the Ad information is created by using a [Create Ad Object](media-api-reference.md#create-ad-object).  For `ChapterStart` event, the Chapter information is created by using a [Create Chapter Object](media-api-reference.md#create-chapter-object).  This is not required for other events. |
+| `info` | For an `AdBreakStart` event, the `adBreak` information is created by using [createAdBreakObject](media-api-reference.md#create-an-adbreak-object) method.  For an `AdStart` event, the Ad information is created by using [createAdObject](media-api-reference.md#create-an-ad-object) method.  For `ChapterStart` event, the Chapter information is created by using [createChapterObject](media-api-reference.md#create-a-chapter-object) method.  This is not required for other events. |
```

Comment:
In all 3 sentences, please add "the" between "using" and "[createAdBreakObject]"